heralding/misc/wordlist.py

```diff
@@ -0,0 +1,10004 @@
+# form SecLists : https://github.com/danielmiessler/SecLists/blob/master/Passwords/xato-net-10-million-passwords-10000.txt
+
+plist = [
```

Comment:
The wordlist should be included as a text file and loaded on startup

Reply:
okay.